homeassistant/components/rympro/sensor.py

```diff
@@ -36,6 +36,20 @@ class RymProSensorEntityDescription(SensorEntityDescription):
         suggested_display_precision=3,
         value_key="read",
     ),
+    RymProSensorEntityDescription(
+        key="monthly_consumption",
+        translation_key="monthly_consumption",
+        state_class=SensorStateClass.TOTAL,
```

Comment:
These sensors should be [`TOTAL_INCREASING`](https://developers.home-assistant.io/docs/core/entity/sensor?_highlight=state_class#state-class-sensorstateclasstotal_increasing). This will also make sure a new billing cycle is automatically detected (except rare cases where there was barely any consumption and the first read the following month is already higher).

Reply:
Done